app/src/main/res/xml/appfilter.xml

```diff
@@ -3422,7 +3534,10 @@
 
     <!-- WiFiKeyShare -->
     <item component="ComponentInfo{be.brunoparmentier.wifikeyshare/be.brunoparmentier.wifikeyshare.ui.activities.WifiListActivity}" drawable="wifikeyshare" />
-
+    
```

Comment:
I think there are a few lines in the diff with extra spaces.

Reply:
I don't fully understand what's happening here. Maybe I removed a space and added another or something like that.